apps/accounts/tests/test_views.py

```diff
@@ -0,0 +1,81 @@
+from django.core.urlresolvers import reverse
+from django.test.utils import override_settings
+
+import mock
+
+from rest_framework.test import APITestCase
+from waffle import Switch
+
+from users.models import UserProfile
+
+FXA_CONFIG = {'some': 'stuff', 'that is': 'needed'}
+
+
+class override_switch(object):
+    def __init__(self, name, active):
+        self.name = name
+        self.active = active
+
+    def __enter__(self):
+        Switch.objects.create(name=self.name, active=self.active)
+
+    def __exit__(self, exc_type, exc_val, exc_tb):
+        Switch.objects.filter(name=self.name).delete()
+
+
+class TestFxALoginWaffle(APITestCase):
+
+    @property
+    def url(self):
+        return reverse('accounts.login')
```

Comment:
couldn't it be `self.url = reverse('accounts.login')` ? The less meta (i.e. `@property`) the better